crates/ra_hir/src/code_model.rs

```diff
@@ -759,6 +764,12 @@ impl MacroDef {
     }
 }
 
+impl From<MacroDef> for ItemInNs {
```

Comment:
These `From` impls should go into a `from_id` module which is a (badly named) place where we interconvert between code-model types and ids.